src/Service.Tests/SqlTests/RestApiTests/Patch/MySqlPatchApiTests.cs

```diff
@@ -210,6 +210,13 @@ public void PatchOneViewBadRequestTest()
         {
             throw new NotImplementedException();
         }
+
+        [TestMethod]
+        [Ignore]
```

Comment:
why Ignore?

Reply:
For MySql, we don't support database policies for upsert/upsertincremental operations. There is an open issue for this already here: https://github.com/Azure/data-api-builder/issues/1267.

Reply:
By not adding this , are you seeing failures? Also please add a comment or update description of #1267 to remember to implement this test when that feature is implemented